content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,108 @@
+If there is no results for the search done, the following message is displayed: "No records found".
+When clicking on "reset", the search is removed and all the results are displayed.
+
+ ## Indexing
+ 
+This section provides information on the number of products that can be searched through shop's search function, and compares it to the number of products present in the catalog.
+
+Firstly an explanation is shown: "The "indexed" products have been analyzed by PrestaShop and will appear in the results of a front office search.
+Indexed products 18 / 18." (the number depends on the products which are in the shop, and are indexed).
+Below folows: "Building the product index may take a few minutes. If your server stops before the process ends, you can resume the indexing by clicking "Add missing products to the index"."
+Below there are 2 links that when pressed performs the following actions:
+ -Add missing products to the index - only the new products will be indexed. Also when pressed, the page will be refreshed, and a message shown: "Successful update."
+ -Re-build the entire index - the "Re-build entire index" process takes more time, but it is more thorough.
+Below it is written: You can set a cron job that will rebuild your index using the following URL: (a full link of the shop and the directory you are in is shown)
+Currently used shop URL which is shown: " http://localhost/presta1781/admin040gp2f9r/index.php?controller=AdminSearch&action=searchCron&ajax=1&full=1&token=yruOjDwR&id_shop=1"
+Below there is a toggle switch for indexing, can be turned "on" or "off". Has an explanation for it: "Enable the automatic indexing of products. If you enable this feature, the products will be indexed in the search automatically when they are saved. If the feature is disabled, you will have to index products manually by using the links provided in the field set."
+
+ ## Search
+ 
+Firsly a message is presented: "We are thrilled to introduce you to the fuzzy search, one of the new features from 1.7.7! Please note that it is still in beta version, so feel free to share improvement ideas on GitHub to have it enhanced."
+Below there are 2 links:
+ - [Signaler un problème sur GitHub](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+ - [Proposer une idée d'amélioration sur GitHub](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+Below are the following:
+ - "Search within word" toggle switch, can be turned to "yes" or "no". Has an explanation below: "By default, to search for “blouse”, you have to enter “blous”, “blo”, etc (beginning of the word) – but not “lous” (within the word). With this option enabled, it also gives the good result if you search for “lous”, “ouse”, or anything contained in the word.". When hovered on the field, a message is shown: "Enable search within a whole word, rather than from its beginning only. It checks if the searched term is contained in the indexed word. This may be resource-consuming.
```

Comment:
```suggestion
 - "Search within word" : This toggle, can be enabled or disabled. The following help text is displayed: "By default, to search for “blouse”, you have to enter “blous”, “blo”, etc (beginning of the word) – but not “lous” (within the word). With this option enabled, it also gives the good result if you search for “lous”, “ouse”, or anything contained in the word.". 
The following tooltip is displayed: "Enable search within a whole word, rather than from its beginning only. It checks if the searched term is contained in the indexed word. This may be resource-consuming."
```

Reply:
What is the default status of this toggle ?

Reply:
FYI, the messages displayed below a field or toggle is called an help text.
And the messages displayed when you hover the label of a field or toggle is called a tooltip.
That's important to specify it.